views/components/Sidebar/SideNavigation.test.jsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import React from 'react'
+import { render, waitFor } from '@testing-library/react'
+import SideNavigation from './SideNavigation'
+import { MemoryRouter } from 'react-router-dom'
+import { createUser } from '../../../test/fixtures'
+
+jest.mock('./SidebarLink')
+
+describe('SideNavigation', () => {
+  let wrapper
+
+  beforeEach(() => {
+    wrapper = render(
+      <MemoryRouter>
+        <SideNavigation user={user} />
+      </MemoryRouter>
+    )
+  })
+  const user = createUser({ role: 'admin' })
+
+  test('renders SideNavigation component', async () => {
+    const { container } = wrapper
+
+    await waitFor(() => {
+      const nav = container.getElementsByClassName('sidenav')
+
+      expect(nav).toBeDefined()
```

Comment:
I think `toBeInTheDocument` is a slight improvement over being defined.

```suggestion
      expect(nav).toBeInTheDocument()
```

Reply:
[6b310d1](https://github.com/AMP-SCZ/dpdash/pull/458/commits/6b310d18b25129974d8ce470610170164ed5d426)